main_test.go

```diff
@@ -194,3 +203,53 @@ func registeredTestWslInstances() ([]wsl.Distro, error) {
 
 	return distros, nil
 }
+
+// defaultDistro gets the default distro's name via wsl.exe to bypass wsl.DefaultDistro in order to
+// better decouple tests.
+func defaultDistro() (string, error) {
+	out, err := exec.Command("powershell.exe", "-Command", "$env:WSL_UTF8=1; wsl.exe --list --verbose").CombinedOutput()
+	if strings.Contains(string(out), "Windows Subsystem for Linux has no installed distributions.") {
+		return "", nil
+	}
+	if err != nil {
+		return "", fmt.Errorf("failed to find current default distro: %v", err)
+	}
+	s := bufio.NewScanner(bytes.NewReader(out))
```

Comment:
niptick: I would add some line breaks in that function by group of functionality :)
```
Command + immediate error check (string contains and err != nil)

Parsing output with scanner

return
```